node/lib/stat.js

```diff
@@ -20,12 +20,398 @@
 
 'use strict';
 
-function Stat(name, type, value, tags) {
+var Cleaner = require('./statsd-clean');
+var clean = Cleaner.clean;
+var cleanHostPort = Cleaner.cleanHostPort;
+
+module.exports = {
+    BaseStat: BaseStat,
+    InboundCallsRecvdTags: InboundCallsRecvdTags,
+    OutboundCallsSuccessTags: OutboundCallsSuccessTags,
+    OutboundCallsLatencyTags: OutboundCallsLatencyTags,
+    OutboundCallsSentTags: OutboundCallsSentTags,
+    OutboundCallsAppErrorsTags: OutboundCallsAppErrorsTags,
+    OutboundCallsPerAttemptLatencyTags: OutboundCallsPerAttemptLatencyTags,
+    OutboundCallsPerAttemptAppErrorsTags: OutboundCallsPerAttemptAppErrorsTags,
+    InboundCallsLatencyTags: InboundCallsLatencyTags,
+    InboundCallsSuccessTags: InboundCallsSuccessTags,
+    InboundCallsAppErrorsTags: InboundCallsAppErrorsTags,
+    InboundRequestSizeTags: InboundRequestSizeTags,
+    ConnectionsBytesRcvdTags: ConnectionsBytesRcvdTags,
+    InboundResponseSizeTags: InboundResponseSizeTags,
+    OutboundRequestSizeTags: OutboundRequestSizeTags,
+    ConnectionsBytesSentTags: ConnectionsBytesSentTags,
+    OutboundResponseSizeTags: OutboundResponseSizeTags
```

Comment:
This begins to resemble errors.js and should probably be moved up a directory to sit beside it.